gc/base/SparseAddressOrderedFixedSizeDataPool.hpp

```diff
@@ -194,6 +195,14 @@ class MM_SparseAddressOrderedFixedSizeDataPool : public MM_BaseVirtual
 		return _freeListPoolAllocBytes;
 	}
 
+	/**
+	 * Get the total count of the allocated Objects
```

Comment:
`Objects` shouldn't be upper-case.
```suggestion
	 * Get the total count of the allocated objects
```